plugins/module_utils/foreman_helper.py

```diff
@@ -692,6 +718,7 @@ def _create_entity(self, resource, desired_entity, params, foreman_spec):
                 The new current state if the entity
         """
         payload = _flatten_entity(desired_entity, foreman_spec)
+        self._validate_supported_payload(resource, 'create', payload)
```

Comment:
The only thing this does is emit the warning, right?

Reply:
Correct.